core/dbt/task/manage.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+from typing import Dict, Set, Tuple
+
+from .compile import CompileTask
+from .runnable import ManifestTask
+from dbt.exceptions import warn_or_error, ValidationException
+from dbt.adapters.factory import get_adapter
+from dbt.contracts.graph.parsed import (
+    ParsedModelNode,
+)
+from dbt.contracts.project import PruneModelsAction
+
+
+class ManageTask(CompileTask):
+    def run(self):
+        ManifestTask._runtime_initialize(self)
+        models_in_codebase = self.manifest.nodes.keys()
```

Comment:
Is there any sense in `ManageTask` accepting `--select` criteria? "Only look in schemas that also contain the selected resources"? I don't _think_ so, but worth a few moments of consideration. Otherwise, we'll expect to always operate over the full set of enables nodes in the manifest.

Reply:
I think we should leave it out for the first version, and add it once we see a real need for it

Reply:
Closing this as a nice to have. In practice I don't expect most teams to use this as drops will probably only be done on the "main" updating run per environment.

Reply:
Agreed! Thanks for giving it a bit of thought. Future nice-to-have, if it's something we'd want to have at all.